web/src/settings_org.ts

```diff
@@ -1268,6 +1268,15 @@ export function register_save_discard_widget_handlers(
             save_organization_settings(data, $save_button, patch_url, success_continuation);
         },
     );
+
+    $container.on(
+        "click",
+        ".subsection-header .subsection-changes-save button",
+        (e: JQuery.ClickEvent) => {
+            e.preventDefault();
+            e.stopPropagation();
```

Comment:
I guess `e.preventDefault` was needed to make sure that default action of form submission does not happen, so `stopPropagation` might not be needed.

Also, a comment explaining why need a handler just for calling this would be good.